apps/bifrost/src/components/events/report/form-responses.tsx

```
@@ -0,0 +1,91 @@
+import YesNoPieChart from "./cards/yes-no-pie-chart";
+
+export default function EventFeedbackFormResponses({
+	preloadedEvent,
+}: Readonly<{ preloadedEvent: Preloaded<typeof api.events.getEvent> }>) {
+	const event = usePreloadedQuery(preloadedEvent);
+
+	if (!event.formId) {
+		return <div>No feedback form available for this event.</div>;
+	}
+
+	const responses = useQuery(api.forms.getFormResponsesByFormId, { formId: event.formId });
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)